Makefile

```diff
@@ -90,6 +92,9 @@ check: $(TEST_EXEC)
 
 check/leaks: $(TEST_EXEC)
 	@bash $(TEST_SCRIPT) memory
+
+check/norm: $(TEST_EXEC)
```

Comment:
is this rule dependent on $(TEST_EXEC)?

Reply:
You are right, norm does not depend on any compilation, it just need to check files no matter what